### PR TITLE
Align README with the one in the template repo

### PR DESCRIPTION
**Pull request by cbartz**
Applicable spec: n/a

### Overview

Align README with the one in the template repo
### Rationale

to ensure a consistent user experience.

### Checklist

- [x] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [x] The documentation is generated using `src-docs`
- [x] The documentation for charmhub is updated
- [x] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,72 @@
...
+[GitHub Runner Webhook Router Documentation](https://charmhub.io/github-runner-webhook-router).
 
-- See the [Juju SDK documentation](https://juju.is/docs/sdk) for more information about developing and improving charms.
+
+## Get started
+As the charm is designed to be used in conjunction with the [GitHub Runner](https://github.com/canonical/github-runner-operator) charm,
+please see [How to set up reactive spawning](https://charmhub.io/github-runner/docs/how-to-reactive) to learn how to deploy the charm.
+
+The charm requires an integration with mongodb (either the machine or k8s charm), otherwise it will
+go into a blocked state.
```
Comment (erinecon) [inline README.md:34]: I would move this line to a section called `## Integrations` (after the "Basic operations" section) and also add a line saying something like

`For a complete list of integrations, see the [Charmhub documentation](https://charmhub.io/github-runner-webhook-router/integrations).`

File: README.md [hunk 2]
```diff
@@ -1,26 +1,72 @@
...
+```
+
+
+## Learn more
+* [Read more](https://charmhub.io/github-runner-webhook-router)
+* [Developer documentation](https://github.com/canonical/github-runner-webhook-router/blob/main/CONTRIBUTING.md)
+* [PaaS Charm](https://github.com/canonical/paas-charm)
+* [Write your first Kubernetes charm for a Flask app](https://juju.is/docs/sdk/write-your-first-kubernetes-charm-for-a-flask-app)
+* [How to build a 12-Factor app charm](https://juju.is/docs/sdk/build-a-12-factor-app-charm)
+
```
Comment (erinecon) [inline README.md:68]: I would include another item in this list, `Troubleshooting`, and include a link to the 12-factor Matrix channel, `https://matrix.to/#/#12-factor-charms:ubuntu.com`

File: README.md [hunk 3]
```diff
@@ -1,26 +1,72 @@
...
+```shell
+juju config github-runner-webhook-router webhook-secret=<your-secret>
+```
+
+
+## Learn more
+* [Read more](https://charmhub.io/github-runner-webhook-router)
+* [Developer documentation](https://github.com/canonical/github-runner-webhook-router/blob/main/CONTRIBUTING.md)
+* [PaaS Charm](https://github.com/canonical/paas-charm)
```
Comment (erinecon) [inline README.md:65]:
```suggestion
* [PaaS Charm repository](https://github.com/canonical/paas-charm)
```
(nitpick)

**Review by erinecon — COMMENTED** [review]
Thanks for updating the README -- it looks great!

**Review by erinecon — APPROVED** [review]
Thanks for updating the README!

**Comment by weiiwang01** [comment]
If you would like to temporarily override the `statsd-exporter` version inside the rock, you can add this section to the end of `rockcraft.yaml`. I will create a pull request in Rockcraft shortly.

```
    flask-framework/statsd-exporter:
      source-tag: v0.28.0
```

**Comment by weiiwang01** [comment]
https://github.com/canonical/rockcraft/pull/770

**Comment by cbartz** [comment]
Thanks @weiiwang01 , I have expected it but still tried it, unfortunately the latest version does not fix the issue as the go library is still outdated.

**Review by varshigupta12 — APPROVED** [review]

**Review by yhaliaw — APPROVED** [review]

**Pull request merged**